pg2kinesis/formatter.py

```diff
@@ -110,24 +111,27 @@ def _preprocess_wal2json_change(self, change):
         changes = []
 
         for change in change_dictionary['change']:
-            table_name = change['table']
-            schema = change['schema']
-            if self.table_re.search(table_name):
-                if self.full_change:
-                    changes.append(FullChange(xid=self.cur_xact, change=change))
-                else:
-                    try:
-                        full_table = '{}.{}'.format(schema, table_name)
-                        primary_key = self.primary_key_map[full_table]
-                    except KeyError:
-                        self._log_and_raise(MISSING_TABLE_ERR.format(full_table))
+
+            if change['kind'].lower() == self.change_kind.lower() or self.change_kind.lower() == 'all':
```

Comment:
Please convert change_kind to lower case once in the `__init__`  above.

Reply:
Is `change['kind']` ever not lower case? This is a hot loop so trying to limit unnecessary operations.

Reply:
`all` should be check first as that is default and will short circuit the next check.

Reply:
Hi @surbas, thanks for the feedback! We've opened a cleaner PR with some tests included here: https://github.com/surbas/pg2kinesis/pull/21